README.md

```diff
@@ -40,12 +43,14 @@ semantics and functions, is available at [PyTorch.org](http://pytorch.org/xla/).
 Google Cloud Platform lets you deploy PyTorch networks running on Cloud TPUs.
 This guide is split into two parts:
 
-* [Running on a Cloud TPU VM](#TPUVM)
```

Comment:
@JackCaoG I merged the TPUVM section into the header section. I think this would be better aligned with the original flow (e.g., how Cloud TPU instructions are split between a single node trainig and a pod training), without losing the content. Let me know what you think.